html/about.html

```diff
@@ -0,0 +1,147 @@
+    <title>About</title>
+    <link rel="stylesheet" href="/css/about.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700&display=swap"
+      rel="stylesheet"
+    />
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+    <script
+      src="https://kit.fontawesome.com/716817a6e7.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+    <header>
+      <div class="page-heading">
+        <i class="fa fa-facebook-f"></i>
+        <i class="fa fa-twitter"></i>
+        <span class="heading-label">English</span>
+        <span class="heading-label">My Page</span>
+        <span class="heading-label">Logout</span>
+      </div>
+      <div class="navbar">
+        <a href="../html/index.html"
+          ><img
+            src="/images/logo.png"
+            alt="Sawasawa brand logo"
+            class="logo-img"
+        /></a>
+        <nav>
+          <div class="nav-links" id="navLinks">
+            <i class="fas fa-times" id="closeBtn"></i>
+            <ul class="nav-items" id="navItems">
+              <li><a href="../html/index.html">About</a></li>
+              <li><a href="#">Program</a></li>
+              <li><a href="#">Join</a></li>
+              <li><a href="#">Sponsor</a></li>
+              <li><a href="#">Trends</a></li>
+            </ul>
+            <button class="btn">Latest Trends</button>
+          </div>
+        </nav>
+        <i class="fas fa-bars" id="openBtn"></i>
+      </div>
+    </header>
+
+    <!-- Headline-Section -->
+    <section class="main-header">
+      <div class="header-section">
+        <span class="text">Hello! Fashion World!</span>
+        <h1 class="about-header">SAWASAWA FASHION WEEK 2022</h1>
+        <p class="header-para">
+          SawaSawa Fashion Week is a yearly event that brings together creatives
+          to meet, network and ofcourse gain more knowledge in the fashion
+          industry. This happens in different countries each year. It was
+          created with a purpose of letting each and every creative share their
+          work and share their opinions freely. However this year we have
+          decided to host it in the country where it all started i.e Nairobi,
+          Kenya. We have decided to add more numbers to the guest list as you
+          know for the past years we only had a particular number of guests.
+          This year will be a new beginning as we target to bring in more ideal
+          speakers and the program entails so much learning in the process.
+        </p>
+        <span
+          >Please contact us for any further queries on Sawasawa Fashion Event
+        </span>
+        <br />
+        <span class="about-email">sawasawafashion2022@sawasawa.org</span>
```

Comment:
Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_211](https://user-images.githubusercontent.com/69638013/169508632-a6e6a0e1-a784-40fb-b890-3a6a943a059f.png) |  ![Screenshot_210](https://user-images.githubusercontent.com/69638013/169508605-f6022142-1f63-46a6-8127-f7aad3adba0a.png)


- [ ] Kindly adjust this content as htey look clustered on mobile version 👍🏿